docs/se/cors/01_introduction.adoc

```diff
@@ -0,0 +1,89 @@
+:model-reader-java: {mp-openapi-prefix}/api/src/main/java/org/eclipse/microprofile/openapi/OASModelReader.java
+:filter-java: {mp-openapi-prefix}/api/src/main/java/org/eclipse/microprofile/openapi/OASFilter.java
+
+Easily allow your Helidon SE application to support link:{cors-spec}[cross-origin resource sharing] (CORS).
+
+== Quick start
+You add CORS support to your application by adding to its <<se/webserver/03_routing.adoc,web server routing set-up>>.
+The following code shows how to prepare your application's routing to support metrics and health support as well as
+CORS.
+
+[[intro-quick-start-code-example]]
+[source,java]
+----
+
+    private static Routing createRouting(Config config) {
+
+        MetricsSupport metrics = MetricsSupport.create();
+        GreetService greetService = new GreetService(config);
+        HealthSupport health = HealthSupport.builder()
+                .addLiveness(HealthChecks.healthChecks())   // Adds a convenient set of checks
+                .build();
+        CorsSupport corsSupport = CorsSupport.builder()  // <1>
+                .addCrossOriginSupport(CrossOriginSupport.builder() // <2>
+                            .allowOrigins("http://foo.com", "http://there.com") // <3>
+                            .allowMethods("PUT", "DELETE") // <4>
+                            .build())
+                .addCrossoriginSupport(CrossOriginSupport.create()) // <5>
+                .build();
+
+        // Note: Add the CORS routing *before* registering the GreetService routing.
+        return Routing.builder()
+                .register(JsonSupport.create())
+                .register(health)                   // Health at "/health"
+                .register(metrics)                 // Metrics at "/metrics"
+                .register("/greet", corsSupport, greetService) // <6>
+                .build();
+    }
+----
+<1> Create a `CorsSupport` instance using its builder.
+<2> Add a `CrossOriginSupport` instance (using _its_ builder) to constrain resource sharing.
+<3> List the origins (sites) allowed to share resources from this app.
+<4> List the HTTP methods the constraint applies to.
+<5> Add a `CrossOriginSupport` instance that permits all sharing (the default).
+<6> Register the new `CorsSupport` instance with -- but in front of -- the service which implements the business logic.
+
+These few additional lines allow the app to participate in CORS.
+
+== Maven coordinates
+
+The <<about/04_managing-dependencies.adoc, Getting Started>> page describes how you
+should declare dependency management for Helidon applications. Then declare the following dependency in your project:
+
+[source,xml,subs="attributes+"]
+----
+<dependency>
+    <groupId>io.helidon.webserver</groupId>
+    <artifactId>helidon-webserver-cors</artifactId>
+</dependency>
+----
+
+See the Helidon CORS support in action by building and running the CORS example link:{helidon-se-cors-example}[here].
```

Comment:
Maybe  mention that you can control cors support via configuration and link to 02_configuration.adoc?

Reply:
Done